external/src/test/java/com/redhat/cloud/policies/engine/process/ReceiverFilterTest.java

```diff
@@ -118,31 +129,55 @@ public void testRejectionTypes() throws Exception {
         receiver.processAsync(Message.of(inputJson)).await().indefinitely();
 
         assertEquals(0, mockedAlertsService.getPushedEvents().size());
-        assertEquals(2, incomingMessagesCount.getCount());
-        assertEquals(0, processingErrors.getCount());
-        assertEquals(2, rejectedCount.getCount());
-        assertEquals(1, rejectedCountReporter.getCount());
-        assertEquals(1, rejectedCountType.getCount());
+        assertEquals(2, incomingMessagesCount.count());
+        assertEquals(0, processingErrors.count());
+        assertEquals(2, rejectedCount.count());
+        assertEquals(1, rejectedCountReporter.count());
+        assertEquals(1, rejectedCountType.count());
     }
 
-
     public static class InternalCounter implements Counter {
 
         private AtomicLong value = new AtomicLong(0);
 
         @Override
-        public void inc() {
+        public void increment() {
             value.incrementAndGet();
         }
 
         @Override
-        public void inc(long l) {
-            value.addAndGet(l);
+        public void increment(double amount) {
+            value.addAndGet(Double.valueOf(amount).longValue());
         }
 
         @Override
-        public long getCount() {
+        public double count() {
             return value.get();
         }
+
+        @Override
+        public Iterable<Measurement> measure() {
+            return Counter.super.measure();
+        }
+
+        @Override
+        public <T> T match(Function<Gauge, T> visitGauge, Function<Counter, T> visitCounter, Function<Timer, T> visitTimer, Function<DistributionSummary, T> visitSummary, Function<LongTaskTimer, T> visitLongTaskTimer, Function<TimeGauge, T> visitTimeGauge, Function<FunctionCounter, T> visitFunctionCounter, Function<FunctionTimer, T> visitFunctionTimer, Function<Meter, T> visitMeter) {
+            return Counter.super.match(visitGauge, visitCounter, visitTimer, visitSummary, visitLongTaskTimer, visitTimeGauge, visitFunctionCounter, visitFunctionTimer, visitMeter);
+        }
+
+        @Override
+        public void use(Consumer<Gauge> visitGauge, Consumer<Counter> visitCounter, Consumer<Timer> visitTimer, Consumer<DistributionSummary> visitSummary, Consumer<LongTaskTimer> visitLongTaskTimer, Consumer<TimeGauge> visitTimeGauge, Consumer<FunctionCounter> visitFunctionCounter, Consumer<FunctionTimer> visitFunctionTimer, Consumer<Meter> visitMeter) {
+            Counter.super.use(visitGauge, visitCounter, visitTimer, visitSummary, visitLongTaskTimer, visitTimeGauge, visitFunctionCounter, visitFunctionTimer, visitMeter);
+        }
+
+        @Override
+        public void close() {
+            Counter.super.close();
+        }
```

Comment:
Is this code really needed? These 4 methods seem to already exist as `default` interface methods in `Meter` and `Counter`.

Reply:
removed